### DISCUSSION TITLE
Transparent Images

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1293] [General]
Is there a config setting to enable transparent images?

I have copied over the config/glide.php and changed a default_params as suggested in another post but this does not work.

**Comment by pboivin** [comment]
Hi @michaeljcoyne, there are a few ways to approach this. Out of the box, the defaut [Glide config](https://github.com/area17/twill/blob/2.x/config/glide.php#L24) will convert all images to JPG.

You can request a specific format for a given image with the `$params` argument :
```blade
<img src="{{ $post->image('cover', 'mobile', ['fm' => 'png']) }}" alt="">
```

Also, there's the [`twill.glide.original_media_for_extensions`](https://github.com/area17/twill/blob/2.x/config/glide.php#L44) configuration that will make sure to return the original uploaded media. This can be useful for GIF, PNG, SVG, for example.
